CMakeLists.txt

```
@@ -472,6 +472,7 @@ if(ISLE_BUILD_LEGO1)
     OUT_TARGETS lego1_targets
   )
   reccmp_add_target(lego1 ID LEGO1)
+  install(TARGETS lego1 DESTINATION bin)
```

I'd just install in `.` here.
The isle repo is purely for decompilation project, and will only ever support Windows.
```suggestion
  install(TARGETS lego1 DESTINATION ".")
```
In isle-portable, I'd use [GNUInstallDirs](https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html) and install to _bindir and _libdir. etc

---

Can you also apply this change to the other `install`'s as well?

---

Yes, I am currently rebasing.